src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/Models/Result.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using System.Text;
 
 namespace DevAdventCalendarCompetition.Repository.Models
 {
-    [Table("Results")]
-    public class Result : ModelBase
+    public class Result
```

Comment:
why have you removed `[Table("Results")]`?

Reply:
Yes, because EF Core by convention will map an entity to a table with the same name as its corresponding DbSet.
It's a good idea to prefer conventions over configuration.

Reply:
correct answer ;)